### DISCUSSION TITLE
how to disable sentry and prevent sending requests

### DISCUSSION THREAD

**Discussion by AlaaDq** [quasarframework/quasar#12602] [General - Components / Directives / etc] [answered]
Request URL: https://o207331.ingest.sentry.io/api/5581756/envelope/?sentry_key=5e74084576624408ad3884553d89628a&sentry_version=7
Request Method: POST
Status Code: 403 
Referrer Policy: origin
![image](https://user-images.githubusercontent.com/45366622/155769081-e007e1fd-d1cd-4288-b347-573ea4b8bf12.png)

**Comment by MilosPaunovic** [comment]
Do you need to disable particular Sentry requests or all of 'em?

**Reply by AlaaDq** [reply]
all request to sentry, please
i am not using it at all but i see a lot of wasted requests to sentry

**Comment by MilosPaunovic** [comment]
You'll need to find the part of code which initializes Sentry and remove/disable it. It's probably in a boot file.

**Reply by AlaaDq** [reply]
there is nothing for Sentry on boot file

**Reply by MilosPaunovic** [reply]
Okay, then you need to find where you initialize Sentry in your project.

**Reply by AlaaDq** [reply]
i didn't  initialize Sentry i think it is come by default with quasar new versions, i am not sure but this is my problem i didn't  initialize Sentry anywhere or use it before in my life

**Comment by yusufkandemir** [comment, marked as answer]
Either you are directly using it in your project, there is a rather malicious package you are using, or you have a browser extension which it. You can check the initiator column to understand the source.

![image](https://user-images.githubusercontent.com/6266078/155839704-69eb51cd-5835-459d-b274-5ac1aafb8889.png)
As an example, the first request on the screenshot is initiated by `refined-github.js`. Then, I clicked on it to confirm where it is coming from:
![image](https://user-images.githubusercontent.com/6266078/155839828-76fb8b23-d1aa-4b87-9daa-6322458d0c36.png)
So, now we are sure that the Refined GitHub extension is making that request.

So, you can find the source of that request by following the same procedure. It is not related to Quasar at all, Quasar or any package/framework that makes sense won't ever configure and ship Sentry or other terms of telemetry, especially without asking consent first.
